docs/reference/glossary.rst

```diff
@@ -0,0 +1,97 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+.. contents::
+  :local:
+
+Flocker release types
+=====================
+
+.. note:: All Flocker documentation will refer the Production ready product, unless tagged Alpha or Beta.
+
+.. _alpha-definition:
+
+Alpha
+   This product or feature is a proof-of-concept; expect API instability; lack of security; product instability.
+
+.. _beta-definition:
+
+Beta
+   This product or feature is approaching readiness; expect minor API instability between versions; incomplete security; product stability for most common uses.
+
+.. _production-ready-definition:
+
+Production ready
+   This product or feature is suitable for production use; expect API stability between versions; robust security; product stability for all supported uses.
```

Comment:
To be more clear: This needs to define "suitable for production use" else it is almost a circular definition.

https://www.google.co.uk/search?q=recursion